integration_tests/grpc_rest/pki/helpers.go

```diff
@@ -1687,6 +1687,21 @@ func Demo(suite *utils.TestSuite) {
 	revDistPoints, _ = GetAllPkiRevocationDistributionPoints(suite)
 	require.Equal(suite.T, 1, len(revDistPoints))
 
+	// Add revocation distribution point for PAA by Vendor with certificate with different whitespaces
+	msgAddPkiRevDistPoints = pkitypes.MsgAddPkiRevocationDistributionPoint{
```

Comment:
Can you please add a dedicated unit and integration tests for the problem (adding a Distr Point for the same certificate as PAA but with whitespaces)? So that it's clear from the tests that we do cover such case.

Reply:
Done